ui/app/AppLayouts/Wallet/popups/AddAccountModal.qml

```diff
@@ -231,6 +215,7 @@ StatusModal {
             StatusColorSelectorGrid {
                 id: colorSelectionGrid
                 anchors.horizontalCenter: parent.horizontalCenter
+                enabled: !d.authenticationNeeded
                 titleText: qsTr("color").toUpperCase()
```

Comment:
Alternativer: `font.capitalization: Font.AllUppercase`

Reply:
@MishkaRogachev not the subject of this PR.